mkosi/distribution/opensuse.py

```diff
@@ -55,7 +56,13 @@ def setup(cls, context: Context) -> None:
     def install(cls, context: Context) -> None:
         packages = ["filesystem"]
         if not any(p.endswith("-release") for p in context.config.packages):
-            packages += ["openSUSE-release"]
+            if context.config.release in ("current", "stable", "leap") or (
+                GenericVersion(context.config.release) >= 16
+                and GenericVersion(context.config.release) < 20000000
```

Comment:
You can shorten this down
```suggestion
                16 <= GenericVersion(context.config.release) < 20000000
```
also the `20000000` comes a bit out of the blue if one doesn't know it's a date (I assume). A comment would help, or maybe
```suggestion
                16 <= GenericVersion(context.config.release) < 2000_00_00
```
though that also only makes the slightly more obvious it's a date.